modular_chomp/code/datums/diseases/advance/symptoms/flesh_eating.dm

```diff
@@ -0,0 +1,42 @@
+/*
+//////////////////////////////////////
+
+Necrotizing Fasciitis (AKA Flesh-Eating Disease)
+
+	Very very noticable.
+	Lowers resistance tremendously.
+	No changes to stage speed.
+	Decreases transmittablity temrendously.
+	Fatal Level.
+
+Bonus
+	Deals brute damage over time.
+
+//////////////////////////////////////
+*/
+
+/datum/symptom/flesh_eating
+	name = "Necrotizing Fasciitis"
+	stealth = -3
+	resistance = -4
+	stage_speed = 0
+	transmittable = -4
+	level = 6
+	severity = 5
+
+/datum/symptom/flesh_eating/Activate(datum/disease/advance/A)
+	..()
+	if(prob(SYMPTOM_ACTIVATION_PROB))
+		var/mob/living/M = A.affected_mob
+		switch(A.stage)
+			if(2,3)
+				to_chat(M, span_warning("[pick("You feel a sudden pain across your body.", "Drops of blood appear suddenly on your skin.")]"))
+			if(4,5)
+				to_chat(M, span_userdanger("[pick("You cringe as a violent pain takes over your body.", "It feels like your body is eating itself inside out.", "IT HURTS.")]"))
```

Comment:
spans are defined as
#define span_type(str) ("<span class='type'>" + str + "</span>") so they can be chained and even procs can be used in them. There's no need to use any with "[proc("")]"

Counts for all uses here. The only important is that the function returns only strings